src/jbmc/jbmc_parse_options.cpp

```diff
@@ -44,7 +44,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <goto-programs/show_symbol_table.h>
 #include <goto-programs/show_properties.h>
 
-#include <goto-symex/adjust_float_expressions.h>
+#include <goto-programs/adjust_float_expressions.h>
```

Comment:
The blank line above that can now go away for it's no longer a separate group/directory.